code/datums/supplypacks/misc_yw.dm

```diff
@@ -0,0 +1,26 @@
+/datum/supply_pack/misc/blueshieldgear
+	name = "Blueshield Special Equipment"
+	contains = list(
+			/obj/item/clothing/head/beret/blueshield = 1,
+			/obj/item/clothing/under/yw/blueshield = 1,
+			/obj/item/clothing/under/yw/blueshield2 = 1,
+			/obj/item/clothing/suit/armor/yw/blueshield = 1,
+			/obj/item/clothing/suit/armor/yw/blueshieldcoat = 1,
+			/obj/item/clothing/accessory/poncho/roles/cloak/blueshield = 1,
+			/obj/item/clothing/accessory/holster/leg = 1
+			)
+	cost = 50
+	containertype = /obj/structure/closet/crate/secure/weapon
+	containername = "Blueshield equipment"
+	access = access_heads
+	
+/datum/supply_pack/misc/blueshieldweapons
+	name = "Blueshield Weapon Kits"
+	contains = list(
+			/obj/item/gunbox/blueshield = 1,
+			/obj/item/gunbox/blueshield/secondary = 1
+			)
+	cost = 50
```

Comment:
```suggestion
	cost = 100
```
Up to 100 this **should** not be needed 99 times out of 100 and if it is the blueshield probably should be demoted or at least reprimanded, plus it discourages anyone else from ordering blueshield gear they should not have their hands on. 

Plus the blueshield gun boxes have access to some powerful stuff and a gun you cant get your hand on any other way.

Reply:
Being 50 sounds very abusable since the blueshield guns are fairly unique and strong

Reply:
Cost increase is definitely reasonable, but I'd argue for different reasons than just the weapons themselves being unique/strong; the cases allow you to pick from a wider range of weapons and they come with a decent amount of free ammo off the bat. Committing now.